### DISCUSSION TITLE
[@types/node] TypeError: Cannot read property 'prototype' of undefined when calling fetch

### DISCUSSION THREAD

**Discussion by YassineOujaa77** [DefinitelyTyped/DefinitelyTyped#69422] [Issues with a @types package]
I'm trying to call fetch API from my react native app ,
this is a test I've done 

`useEffect(() => {
        fetch('https://fakestoreapi.com/products/1')
            .then((res) => res.json())
            .then((json) => console.log(json))
            .catch((err) => console.log('err', err));
    }, []);
`

but I get this error : TypeError: Cannot read property 'prototype' of undefined

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
This is a runtime error and is not related to `@types/node` in any way. Raise this issue on https://github.com/nodejs/node repo
